lalrpop-util/src/lib.rs

```diff
@@ -108,14 +118,32 @@ impl<L, T, E> ParseError<L, T, E> {
         }
     }
 
+    /// Transform a ParseError by applying a function to the location field
+    ///
+    /// This could be useful for example to obtain a `ParseError` where all
+    /// fields implement `Display` or `Debug`.  `ParseErrors` where all fields
+    /// implement `Display` also implement `Display` and `ParseErrors` where all
+    /// fields implement both `Display` and `Debug` implement `Error`
     pub fn map_location<LL>(self, op: impl FnMut(L) -> LL) -> ParseError<LL, T, E> {
         self.map_intern(op, |x| x, |x| x)
     }
 
+    /// Transform a ParseError by applying a function to the token field
```

Comment:
For the enum, each sentence ended with a period, maybe the first line here should as well for consistency